docs/config.rb

```diff
@@ -61,6 +61,17 @@
 # Helpers
 ###
 
+
+config[:credentials] = {
+  documentation: {
+    app_id: 'plFMJJT5O9PC',
+    api_key: '8b126ce956636c64b6e74c8b3f3d0e5e'
+  },
```

Comment:
Not sure if we will go to that step yet, as it's still interesting for dev to quickly get it setup.
But at least, we will have a better visibility on the usage of our users, and it will unlock them the non-authenticated limits.